### PR TITLE
INT-2957: manage attribute `id`

### PR DESCRIPTION
**Pull request by lorenzo-pomili**
INT-2957
issue: https://github.com/tinymce/tinymce-webcomponent/issues/32

### CODE REVIEW

**Review by spocke — APPROVED** [review]
Should this also have a changelog item?

**Review by vpyshnenko — APPROVED** [review]

**Comment by lorenzo-pomili** [comment]
@spocke I added the changelog entry, have a look at it

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Now the `id` attribute is correclty managed
```
Comment (spocke) [inline CHANGELOG.md:10]: Maybe this:

```suggestion
- The `id` attribute was not being used as the id for the editor.
```

Comment (lorenzo-pomili) [reply CHANGELOG.md]: fixed

**Pull request merged**